rerun_py/rerun_sdk/rerun/log/lines.py

```diff
@@ -93,6 +94,8 @@ def log_line_strip(
 
     if positions is not None:
         positions = np.require(positions, dtype="float32")
+        if not positions.any() and len(positions.shape) == 1:
```

Comment:
`positions.any()` is weird.

From [the docs](https://numpy.org/doc/stable/reference/generated/numpy.any.html): "Test whether any array element along a given axis evaluates to True.". I don't think you should be testing positions for `True`